Bio/Restriction/Restriction.py

```diff
@@ -258,6 +258,9 @@ def __init__(cls, name='', bases=(), dct=None):
         # super(RestrictionType, cls).__init__(cls, name, bases, dct)
         try:
             cls.compsite = re.compile(cls.compsite)
+        except AttributeError:
+            # Can happen if initialised wrongly.
+            pass
```

Comment:
Should we add in the comment the same explanation as below ('... Sphinx trying to inspect...')?